modules/ROOT/pages/apikit-odatav4-extension-reference.adoc

```diff
@@ -1,82 +1,116 @@
-| Name | String | The name for this configuration. Connectors reference the configuration with this name. | | *x*
-| Api Definition a| String |  API definition |  | *x*
-| Name a| String |  The identifier of this element used to reference it in other components |  | *x*
+| Name | String | Name for this configuration. Connectors reference the configuration with this name | | *x*
+| Api Definition | String | API definition |  | *x*
+| Name | String | Identifier of this element used to reference it in other components |  | *x*
 |===
 
 ==== Associated Operations
-* <<Route>> 
-* <<SerializeEntity>> 
-* <<SerializeEntityCollection>> 
-* <<TransformToSqlSelect>> 
+
+* xref:apikit-odatav4-extension-reference#route[]
+* xref:apikit-odatav4-extension-reference#serialize-entity[]
+* xref:apikit-odatav4-extension-reference#serialize-entity-collection[]
+* xref:apikit-odatav4-extension-reference#transform-to-sql-select[]
 
 ==== Associated Sources
-* <<RequestEntityCollectionListener>> 
-* <<RequestEntityListener>> 
+
+* xref:apikit-odatav4-extension-reference#on-entity-collection-request[RequestEntityCollectionListener]
+* xref:apikit-odatav4-extension-reference#on-entity-request[RequestEntityListener]
 
 == Operations
 
-[[Route]]
 == Route
+
 `<apikit-odata:route>`
 
 === Parameters
 
-[%header%autowidth.spread]
+[cols="1a,2a,3a,4a,5a", options="header"]
 |===
 | Name | Type | Description | Default Value | Required
-| Configuration | String | The name of the configuration to use. | | *x*
-| Content a| Any |  |  #[payload] | 
-| Output MIME Type a| String |  The MIME type of the payload that this operation outputs. |  | 
-| Output Encoding a| String |  The encoding of the payload that this operation outputs. |  | 
-| Config Ref a| ConfigurationProvider |  The name of the configuration to be used to execute this component |  | *x*
-| Streaming Strategy a| * <<RepeatableInMemoryStream>>
-* <<RepeatableFileStoreStream>>
-* non-repeatable-stream |  Configure if repeatable streams should be used and their behaviour |  | 
-| Method a| Enumeration, one of:
-
-** GET
-** POST
-** PUT
-** PATCH
-** DELETE |  HTTP request method name. |  | *x*
-| Masked Request Path a| String |  OData request path, relative to listener. |  | *x*
-| Scheme a| Enumeration, one of:
-
-** HTTP
-** HTTPS |  HTTP request scheme. |  | *x*
-| Host a| String |  Request host. |  | *x*
-| Listener Path a| String |  Listener path, ending with "*". |  | *x*
-| Http Headers a| Object |  HTTP request headers. |  | 
-| Query String a| String |  HTTP query string. |  | 
-| Target Variable a| String |  The name of a variable on which the operation's output is be placed. |  | 
-| Target Value a| String |  An expression that is evaluated against the operation output and the outcome of that expression is stored in the target variable. |  #[payload] | 
+| Configuration | String | Name of the configuration to use | | *x*
+| Content | Any |  |  #[payload] | 
```

Comment:
Global comment: for consistency, all code values and functions/operations like `#[payload]`, `WHERE`, `$skip`, `$search`, `true`, `false`, etc. should be in backticks.

Reply:
OK! Thanks!